### ISSUE TITLE
Engineering borg side displacement is weird

### ISSUE THREAD

**Issue by Ghagliiarghii** [space-wizards/space-station-14#29977]
## Description

When putting hats on an engineering cyborg, the hats appeared correctly from the front but weird on the side (using wrong displacement map)
**Reproduction**

Place hat on borg, walk the borg to the left or right
**Screenshots**

![image](https://github.com/user-attachments/assets/d5947385-19e2-42b1-9f90-c850bb06843a)

**Additional context**

**Comment by deltanedas** [comment]
inventory offsets dont support directions and its not using displacement

**Comment by lzk228** [comment]
borgs should be able to wear only top hats (lika caps, hardhats etc.)

**Comment by Ghagliiarghii** [comment]
> inventory offsets dont support directions and its not using displacement

Oh yeah you're right... I don't know how to fix it then other than just making a custom sprite for every hat just for engi borg which sounds like a giant mess / hard to support / bad precedent

**Comment by IProduceWidgets** [comment]
probably just needs displacement maps now?

**Comment by deltanedas** [comment]
yes thats the solution, make displacement maps for every borg and remove the inventory offsets, just have a borg inventory layout that specifies a slot and thats it

could even have something like making hats bend over a round head instead of having to sit flat
